templates/extras/calc.md

```diff
@@ -52,65 +30,70 @@ proof is complete. Use `_` as in the example above or `sorry` to stand
 for missing justifications. `sorry` will supress error messages
 entirely, while `_` will generate a guiding error message.
 
-If the structure of calc is incorrect (e.g., missing `:` or the
+If the structure of calc is incorrect (e.g., missing `:=` or the
 justification after it), you may see error messages that are obscure
-and/or red squiggles that end up under a random `...`. To avoid these,
+and/or red squiggles that end up under a random `_`. To avoid these,
 you might first populate a skeleton proof such as:
```

Comment:
I'm not actually sure this is true anymore. You should test precisely what happens in several common scenarios (missing `_` after the `:=`, missing `:=`, missing `_` before the relation...) and report back.